consensus/core/src/network/tonic_network.rs

```diff
@@ -295,7 +296,7 @@ impl ChannelPool {
         let buffer_size = config.connection_buffer_size;
         let endpoint = tonic::transport::Channel::from_shared(address.clone())
             .unwrap()
-            .connect_timeout(timeout)
+            .connect_timeout(CONNECT_TIMEOUT)
```

Comment:
I'm a bit unsure about the `1s` value. Validators may need to tolerate higher latencies at the start of an epoch. How about only use 1s timeout in simtests, or we can wait and see the if the simtest behavior will get updated?

Reply:
It seems like we do want a lower value though right? If we don't use a timeout that is less then the timeout passed in we will not retry in the loop below.

Reply:
@arun-koshy I think that might be ok for the retry loop, as this covers all the error cases. If the endpoint can't connect and times out I guess it should still be ok. I think as @mwtian using a lower `msim` value might be ok.

Reply:
To make sure we are on the same page:
1. I assume in real environments timeouts only happen on misconfigured hosts or when network condition is bad. At the start of an epoch when consensus has not started on all nodes, trying to connect to consensus at peer when it is not started should not need to wait for timeouts.
2. Not just connections, any request can timeout if the peer restarts or the response packet drops. The caller needs to decide the retry strategy. The connection retry loop is best effort in trying to handle some simple connection errors. But the caller is responsible to retry timeouts here.

Another alternative is to use exponentially larger intervals from 1s to 10s on retries, if it is straightward.